interface/weno/formularyInsert.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * weno rx formulary update.
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+ require_once("../globals.php");
+ 
+ echo xlt("Trying")."<BR>";
+
+$zip = new ZipArchive;
+$zip->open('FormularyFiles.zip');
+$zip->extractTo($GLOBALS['temporary_files_dir']);
+$zip->close();
+
+ $file = $GLOBALS['temporary_files_dir'] . "./Weno Exchange LLC/BSureFormulary.txt";
+ $lines = count(file($file));
+ echo text($lines). "<br>";
+  
+ $i = 2;
+
```

Comment:
To drastically speed up this import of data, place the following lines here:
``````
// Settings to drastically speed up import with InnoDB
sqlStatementNoLog("SET autocommit=0");
sqlStatementNoLog("START TRANSACTION");
````

Reply:
still need to address this here
(also note the note I'll repost below that should be done after below code snipper)
(this will drastically increases the speed of this large import)

Reply:
@juggernautsei ,
Still need to address this. see my post below for details.

Reply:
Added